kivy/gesture.py

```diff
@@ -252,6 +341,12 @@ def __init__(self, tolerance=None):
         else:
             self.tolerance = tolerance
 
+        if name is not None:
+            self.name = name
```

Comment:
Better to define `name`, even if a name is not supplied, so the user can access the name value without raising an error.

Reply:
Done